powerdns/models/powerdns.py

```diff
@@ -723,3 +727,39 @@ class Meta:
 
     def __str__(self):
         return self.domain
+
+
+
+try:
+    import pyhermes
+except ImportError:
+    pass
+else:
+    if settings.TXT_AUTOUPDATE_TOPIC_NAME:
+        @transaction.atomic
+        @pyhermes.subscriber(topic=settings.TXT_AUTOUPDATE_TOPIC_NAME)
+        def _update_auto_txt(data):
+            """
+            Auto update txt records depending on pyhermes utility.
+
+            data = [{
+                'name': 'www.example.com',
+                'type': 'TXT',
+                'content': 'new or update value',
+                'subtype': 'SOME_TYPE',
+            }, ..]
+            """
+            #TODO:: migrate data from old to new schema
+            for record_data in data:
+                domain = ??
+                Record.objects.update_or_create(
```

Comment:
Then this is getting much closer to api/v2/Record endpoint (in terms of included functions).
And if so, then question appears.. could we do it less redundant?